src/main/java/com/bootcamp/demo/restapi/dto/battery/BatteryRequestDTO.java

```diff
@@ -0,0 +1,52 @@
+package com.bootcamp.demo.restapi.dto.battery;
+
+import com.google.common.base.Objects;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class BatteryRequestDTO {
+
+    @NotNull(message = "Please provide the battery level")
+    @Min(value = 0, message = "The battery level minimum value should be 0 ")
+    @Max(value = 100, message = "The battery level maximum value should be 100")
+    private Double level;
+
+    public Double getLevel() {
+        return level;
+    }
+
+    public void setLevel(Double level) {
+        this.level = level;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        final var battery = (BatteryRequestDTO) o;
+
+        return Objects.equal(this.getLevel(), battery.getLevel());
```

Comment:
What is the advantage of using com.google.common.base.Objects.equal vs java.util.Objects.equals or is this just a wrong import?